src/flows/context/flow.query.tsx

```diff
@@ -242,39 +245,27 @@ export const FlowQueryProvider: FC = ({children}) => {
     Promise.all(
       map
         .reduce((acc, curr) => {
-          acc.push({
-            text: curr.text,
-            instances: curr.instances.filter(i => !i.modifier).map(i => i.id),
-          })
-
```

Comment:
why were we doing this to begin with with the concat underneath?

Reply:
in the before time, we used to join the data requirements of panels together, and send the requests as one. now each panel seems to make a slight modification to the query to scope the results it wants. simpletable adds a limit, visualization adds an aggregate, etc. So instead of doing the join (where things that dont define a query get squished together) those things just... dont get data.